src/agent/Cargo.lock

```diff
@@ -88,9 +88,9 @@ dependencies = [
 
 [[package]]
 name = "anyhow"
-version = "1.0.68"
+version = "1.0.69"
```

Comment:
Thanks @Xynnn007. Would it be possible to split your `Cargo.lock` changes into two parts, one that derives from build updates (which arguably you don't even need to commit) and the part that specifically adds the required `ttrpc` support? That would make it easier to review.

Reply:
Great suggestion! Let me fix this and then force push with other change altogether